Header header
int64 vehicleID
float64 posx
float64 posy
float64 posz
float64 velx
float64 vely
float64 velz
float64 attyaw
float64 attpitch
float64 attroll
float64 attq0
float64 attq1
float64 attq2
float64 attq3
float64 angvelx
float64 angvely
float64 angvelz
